recipes/orange3-fairness/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "Orange3-Fairness" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18f85525bf1a4b1cf6286a92051b6053ad745826311c037cd40482411b95824b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - aif360 ==0.5.0
+    - numpy 1.23.*
+    - orange3
+    - pip
+    - python >=3.9
+    - setuptools
+  run:
+    - aif360 ==0.5.0
+    - numpy 1.23.*
+    - orange3
+    - python >=3.9
+    - setuptools
+
+test:
+  imports:
+    - orangecontrib
+    - orangecontrib.fairness
+    - orangecontrib.fairness.evaluation
+    - orangecontrib.fairness.modeling
+    - orangecontrib.fairness.widgets
+    - orangecontrib.fairness.widgets.tests
+
```

```suggestion
test:
  imports:
    - orangecontrib
  commands:
    - pip check
  requires:
    - pip

```
You can't have elaborate test with orange packages because of the absence of any graphical interface.